src/style.css

```diff
@@ -2,0 +1,46 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(../src/images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
```

Comment:
The selector `.stars--0 .stars__star:nth-child(-n + 0)` is logically incorrect as it selects no elements. Consider revising this to achieve the desired effect.

Reply:
Hello. I fixed everything, but I can't sync again with GitHab because @import in vscode doesn't work for me. How to fix it?